rules/windows/credential_access_browsers_unusual_parent.toml

```diff
@@ -0,0 +1,125 @@
+- Collect forensic artifacts (process memory, browser profiles, injected modules) for further investigation and potential IOCs.
+- Reset credentials for accounts that may have been exposed through the compromised browser session.
+- Deploy updated endpoint protection signatures and enable stricter application control policies to prevent browsers from being launched by untrusted processes.
+- Enhance monitoring for browser processes launched with debugging flags or code injection indicators across the environment.
+- Escalate to the SOC or IR team to determine whether this event is part of a larger credential theft campaign or linked to other lateral movement activity."""
+references = ["https://www.elastic.co/security-labs/katz-and-mouse-game"]
+risk_score = 73
+rule_id = "46b01bb5-cff2-4a00-9f87-c041d9eab554"
+severity = "high"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Credential Access",
+    "Data Source: Elastic Endgame",
+    "Data Source: Elastic Defend",
+    "Data Source: Windows Security Event Logs",
+    "Data Source: Microsoft Defender for Endpoint",
+    "Data Source: Sysmon",
+    "Data Source: SentinelOne",
+    "Resources: Investigation Guide",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+  process.name : ("chrome.exe", "msedge.exe") and
+  process.parent.executable != null and process.command_line != null and
+  (
+  process.command_line :
+           ("\"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe\"",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\"",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" --headless --disable-logging --log-level=3 --v=0",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" --headless --log-level=3",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" --headless",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" --remote-debugging-port=922? --profile-directory=\"Default\"*",
+            "\"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" --headless --restore-last-session --remote-debugging-port=45452*") or
+
+  (process.args : "--remote-debugging-port=922?" and process.args : "--window-position=-*,-*")
+   ) and
+  not process.parent.executable :
+                         ("C:\\Windows\\explorer.exe",
+                          "C:\\Program Files (x86)\\*.exe",
+                          "C:\\Program Files\\*.exe",
+                          "C:\\Windows\\System32\\rdpinit.exe",
+                          "C:\\Windows\\System32\\sihost.exe",
+                          "C:\\Windows\\System32\\RuntimeBroker.exe",
+                          "C:\\Windows\\System32\\SECOCL64.exe")
```

Comment:
This would be very noisy in CS, as they don't populate process.parent.executable